### ISSUE TITLE
inspect.getargspec(json.load) break with Python 3.6

### ISSUE THREAD

**Issue by Natim** [stefankoegl/python-json-patch#57]
```
jsonpatch.py:108: in get_loadjson

    argspec = inspect.getargspec(json.load)
```

**ValueError: Function has keyword-only parameters or annotations, use getfullargspec() API which can support them**

**Comment by Natim** [comment]
This error comes from jsonpatch==1.14, it seems the code have changed in master.

**Comment by jhamhader** [comment]
@stefankoegl this issue affects warlock and through it glanceclient from OpenStack. Can you please make a release?

**Comment by stefankoegl** [comment]
I have just released version 1.15 which should fix the issue. Please let me know if this solves your problem.
